capabilityquerier/pkg/capabilityquerier/capabilityquerier_integration_test.go

```diff
@@ -3,35 +3,47 @@
 package capabilityquerier_test
```

Comment:
I think this should be package capabilityquerier instead of capabilityquerier_test
Then you don't need to import the capabilityquerier or duplicate the capabilityStatement() and testClientWithContentType(contentType string) functions.

I looked at the history of this file and it was always in it's own package, but I can't think of why this would be the exception.

Reply:
I think you're right, I'm betting that it's from when we were still figuring out how go worked, I'll update it!